test/brod_SUITE.erl

```diff
@@ -0,0 +1,88 @@
+%%%
+%%%   Copyright (c) 2019, Klarna Bank AB (publ)
+%%%
+%%%   Licensed under the Apache License, Version 2.0 (the "License");
+%%%   you may not use this file except in compliance with the License.
+%%%   You may obtain a copy of the License at
+%%%
+%%%       http://www.apache.org/licenses/LICENSE-2.0
+%%%
+%%%   Unless required by applicable law or agreed to in writing, software
+%%%   distributed under the License is distributed on an "AS IS" BASIS,
+%%%   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%%%   See the License for the specific language governing permissions and
+%%%   limitations under the License.
+%%%
+
+%% @private
+-module(brod_SUITE).
+
+%% Test framework
+-export([ init_per_suite/1
+        , end_per_suite/1
+        , all/0
+        , suite/0
+        ]).
+
+%% Test cases
+-export([ t_create_topics/1
+        , t_delete_topics/1
+        , t_delete_topics_not_found/1
+        ]).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/assert.hrl").
+
+-define(HOSTS, [{"localhost", 9092}, {"localhost", 9192}]).
```

Comment:
Remove the second host, I believe it's the reason behind tiimeouts in CI.

Reply:
done but the issue is still occuring